src/onnx/parse_slice.cpp

```diff
@@ -77,7 +86,39 @@ struct parse_slice : op_parser<parse_slice>
             op.axes = axes;
         }
 
-        return info.add_instruction(op, args[0]);
+        if(reverse_direction)
+        {
+
+            migraphx::argument axes_arg = args.at(3)->eval();
+            std::vector<int> axes_v;
+            axes_arg.visit([&](auto s) { axes_v.assign(s.begin(), s.end()); });
```

Comment:
Only the axes that have a negative step should be copied to the reverse operator:

```cpp
std::vector<int64_t> raxes;
for(auto i:range(op.axes))
{
    if (steps[i] < 0)
        raxes.push_back(op.axes[i]);
}
```

Reply:
Done